src/Components/JoinScreen/Chatbox.tsx

```diff
@@ -0,0 +1,61 @@
+import { ChangeEvent, ReactElement, useEffect, useState } from "react";
+
+import { Button } from "@mui/material";
+import socket from "../Websocket/socket";
+import { useParams } from "react-router-dom";
+
+const Chatbox = (): ReactElement => {
+  const {roomName} = useParams()
+  const [messages, setMessages] = useState<string[]>([])
+  const [roomExists, setRoomExists] = useState(false)
+  const [currentMessage, setCurrentMessage] = useState("")
+  
+  useEffect(()=>{
+    socket.emit("get_room", roomName)
+    socket.on("send_room", (room)=>{
+      if(room){
+        if(!roomExists) setRoomExists(true)
+      }
+    })
+    socket.on("message_to_client", (message)=>{
+      console.log(message, 'new Message', messages)
```

Comment:
We should probably come up with a system on when we want to keep console logs before merging. For example, I think it makes sense to have the console log to show that we connected to a server in the backend. Console logs like this that were used for debugging should probably be deleted.